habitat-baselines/habitat_baselines/rl/multi_agent/multi_agent_access_mgr.py

```diff
@@ -175,17 +178,21 @@ def _sample_active(self):
             replace=self._pop_config.allow_self_play,
         )
 
-        self._multi_storage.set_active(
-            [self._agents[i].rollouts for i in self._active_agents]
-        )
-        self._multi_updater.set_active(
-            [self._agents[i].updater for i in self._active_agents]
-        )
+        if not self._is_post_init:
+            # If not post init then we are running in evaluation mode and
```

Comment:
It seems that the code does `if NOT _is_post_init then initialize storage`, but the comment you added says that `if NOT _is_post_init then DON'T initialize`. I may be missing something, could you clarify ?


This feels strange. You are assuming how the code will be called. I would log a warning here.
"_get_agents was called before post_init. As such, only the policy will be setup and not the rollout storage. This is expected in evaluation mode"

Reply:
Oops, yeah this was a mistake.